cancer_ai/validator/model_db.py

```diff
@@ -130,38 +125,6 @@ def update_model(self, chain_miner_model: ChainMinerModel, hotkey: str):
         finally:
             session.close()
 
-    def get_block_timestamp(self, block_number):
```

Comment:
just making sure - this is removed because we check the dates when downloading models from hf and comparing with the db insert date?

Reply:
yes